pkg/broker/ingress/handler.go

```diff
@@ -164,7 +164,7 @@ func (h *Handler) ServeHTTP(response nethttp.ResponseWriter, request *nethttp.Re
 	defer cancel()
 	defer func() { h.reportMetrics(request.Context(), broker, event, statusCode) }()
 	if res := h.decouple.Send(ctx, broker, *event); !cev2.IsACK(res) {
-		msg := fmt.Sprintf("Error publishing to PubSub for broker %s. event: %+v, err: %v.", broker, event, res)
+		msg := fmt.Sprintf("Error publishing to PubSub for broker %s: %v.", broker, res)
```

Comment:
We should be logging using the DesugaredLogger and fields throughout the handler. That can be done in a followup PR.